### ISSUE TITLE
Dimmer setting mode

### ISSUE THREAD

**Issue by erikvanderhorst82** [robertklep/homebridge-klikaanklikuit#13]
I'm running the kaku plugin in my raspberry pi with a 433mhz transmitter. It works almost perfect. I have a lot of dimmers. And when the lights are on and when I ask siri to put all the lights on. All the lights that allready are on are going into dimmer setting mode. So from 100% to 0% and so on.... is there a fix for that?

**Comment by kevinriemens** [comment]
I fixed a simulator issue in a fork by having the ON command for dimmable device prevented, and instead sending a "dim to 100%" command.

**Comment by erikvanderhorst82** [comment]
Yes I also did this, but when I put a light on with the lightswitch on the wall for example for 100%. And then give the command with siri put all the lights on in the livingroom. The one that was alleready on goes in the dinmer setting mode.

**Comment by robertklep** [comment]
@erikvanderhorst82 did you set the "dimmable" option for those devices in the Homebridge configuration? See the example [here](https://github.com/robertklep/homebridge-klikaanklikuit#configuration).

If you enable that option, the plugin will try and prevent multiple "on" commands being sent to the device (although this may not work reliably if you also use a regular KaKu remote to handle the device state).

@kevinriemens feel free to create a pull request against my repo, it sounds like a useful solution!

**Comment by erikvanderhorst82** [comment]
Yes I did. And I'm using the dimmers that you can build in your original light switch. So the problem is that the kaku doesn't give feedback if its on or of?

**Comment by kevinriemens** [comment]
@erikvanderhorst82  KAKU is one way, so does not provide any feedback about it's on or off state. It can only receive commands.

@robertklep will do!
